lib/apm.js

```diff
@@ -9,7 +9,7 @@ class Instrumentation extends EventEmitter {
   instrument(MongoClient, callback) {
     // store a reference to the original functions
     this.$MongoClient = MongoClient;
-    const $prototypeConnect = this.$prototypeConnect = MongoClient.prototype.connect; // eslint-disable-line
+    const $prototypeConnect = (this.$prototypeConnect = MongoClient.prototype.connect); // eslint-disable-line
```

Comment:
This was probably just changed when you ran `npm run format`, but I'm okay with leaving this fix if @daprahamian is! If you do leave this, remove the comment.

Reply:
Since it has been annoying us from the start, I say we leave it in, and remove the eslint comment.